book.rb

```diff
@@ -0,0 +1,19 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :retnals
```

Comment:
- [ ] There is a typo in the `rentals` here, please fix this and also it should be present in the `attr_accessor` itself, so change the scope for rentals as well.